server/routes/api.ts

```diff
@@ -274,7 +275,15 @@ router.get('/help-centre/topic/:topic', getTopicHandler);
 
 router.post('/contact-us', contactUsFormHandler);
 
-router.post('/reminders', createReminderHandler);
+router.post('/reminders/create', createOneOffReminderHandler); // requires sign-in
+router.post(
+	'/reminders/create-public/one-off',
+	publicCreateReminderHandler('ONE_OFF'),
+); // does not require sign-in, uses verification token
+router.post(
+	'/reminders/create-public/recurring',
+	publicCreateReminderHandler('RECURRING'),
+); // does not require sign-in, uses verification token
```

Comment:
Feel free to ignore this, it's very nitpicky! I like using the RESTful pattern of using the HTTP method and the URL path to describe the action - i.e. POST = create (the verb) and `/reminders` = the noun. So the `/create` in `/reminders/create` feels slightly redundant as the POST already conveys that information. For the new endpoints, perhaps `/public` at the start of the path could be a way of scoping the endpoints which don't require sign-in, leaving us with `POST /public/reminders/one-off` and `POST /public/reminders/recurring` (or even `/public/one-off-reminders` and `/public/recurring-reminders`. What do you reckon? As I said do feel free to ignore!

Reply:
The reason I like having `create` in the path is that we also have:
`POST /reminders/cancel`
`POST /reminders/reactivate`

So `POST` feels a bit ambiguous.
I do however like your idea of scoping the endpoints to `/public/...`, as it's an important difference, so I'll make that change